static/app/gettingStartedDocs/python/aiohttp.tsx

```diff
@@ -94,56 +81,47 @@ const onboarding: OnboardingConfig = {
           language: 'python',
           code: `
 ${getSdkSetupSnippet(params)}
-async def hello(request):
-   return web.Response(text="Hello, world")
+###the following lines of code
+###are for testing
+async def debug(request):
+   1/0  # raises an error
+   return web.Response(text="Debug with Sentry")
 
 app = web.Application()
-app.add_routes([web.get('/', hello)])
+###this one too
+app.add_routes([web.get('/sentry-debug', sentry-debug)])
```

Comment:
instead of having 3x comments

![image](https://github.com/user-attachments/assets/68ab9754-bf33-49b5-a086-9faf82300f56)


shouldn't we do like:

```
### Start block: Testing code

async def debug(request):
    1/0  # Raises an error for testing
    return web.Response(text="Debug with Sentry")

app = web.Application()

app.add_routes([web.get('/sentry-debug', debug)])

### End block: Debugging setup
```